packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { Heading } from "../Heading";
+import { Badge } from "../Badge";
+import { Icon } from "../Icon";
+
+export enum InteractiveElementType {
+  Card = "card",
+}
+
+export type UtilityCardProps = {
+  /** Sets the card image source */
+  imageAlt: string;
+  /** Sets the card image alt */
+  imageSrc: string;
+  /** Sets the title to be rendered as h2 */
+  title: string;
+  /** Sets the type of the clickable element */
+  interactiveElementType?: InteractiveElementType;
+  /** Sets the href to be added to the interactive element */
+  href?: string;
+  /** Text describing the kind of service provided */
+  serviceTag: string;
+  /** Text describing the current status of the service as a badge */
+  status?: string;
+  /** Used by StyledComponents to render the component as a specific tag. If href is passed it'll be rendered as "a" */
+  as?: React.ComponentProps<typeof Box.div>["as"];
+  /** Callback function to be used to invoke onClicks */
+  onClick?: React.MouseEventHandler;
+};
+
+const cardBg = {
+  _: "colorBackgroundWeakest",
+  focus: "colorBackgroundWeakest",
+};
+
+const interactiveBg = {
+  active: "colorBackgroundWeak",
+  hover: "colorBackgroundWeak",
+  ...cardBg,
+};
+
+export const UtilityCard: React.FC<UtilityCardProps> = ({
+  as = "div",
+  imageAlt,
+  imageSrc,
+  interactiveElementType,
+  href,
+  title,
+  serviceTag,
+  status,
+  onClick,
+}) => {
+  const isCardInteractive =
+    InteractiveElementType.Card === interactiveElementType;
+
+  const interactiveElementProps = {
+    href,
+    onClick,
+    target: "_blank",
+    rel: "noopener noreferrer",
+    cursor: "pointer",
+  };
+
+  return (
+    <Box.div
+      as={isCardInteractive ? "a" : as}
+      backgroundColor={isCardInteractive ? interactiveBg : cardBg}
+      borderRadius="borderRadius30"
+      display="flex"
+      flexDirection={{ _: "column", lg: "row" }}
+      padding="space50"
+      textDecoration="none"
+      {...(isCardInteractive ? interactiveElementProps : {})}
+    >
+      <Box.div
+        alignItems="center"
+        backgroundColor="bgPrimary"
+        display="flex"
+        h={{ _: "56px", md: "56px" }}
+        justifyContent="center"
+        marginRight="space50"
+        minWidth={{ _: "24px", md: "24px" }}
+        padding={{ _: "space50", md: "space70" }}
+        w={{ _: "56px", md: "56px" }}
+      >
+        <Box.img alt={imageAlt} aria-hidden h="24px" src={imageSrc} w="24px" />
```

Comment:
```suggestion
        <Box.img decorative alt="" aria-hidden h="24px" src={imageSrc} w="24px" />
```

Reply:
I'll leave decorative out, as that's not an accepted prop for Box.img